API/models/paymentDao.js

```diff
@@ -35,6 +36,126 @@ const buyingAddress = async (addressId, userId, biddingId) => {
   }
 };
 
+const sellBidding = async (userId, biddingId) => {
+  try {
+    return await appDataSource.query(
+      `
+      SELECT
+        pi.url AS productImage,
+        s.bid_price AS bidPrice,
+        d.selling_commission AS commission,
+      DATE_FORMAT(s.due_date, '%Y-%m-%d') dueDate
+      FROM sellings s
+      JOIN product_images pi ON pi.product_id = s.product_id
+      JOIN deals d ON d.selling_id = s.id
+      WHERE s.id = ?
+      AND user_id = ?
+      `,
+      [biddingId, userId]
+    );
+  } catch (err) {
+    console.log(err);
+    throw new DatabaseError('DATABASE_ERROR');
+  }
+};
+
+const sellbiddingInfoId = async (
```

Comment:
이 함수의 역할이 뭘까요?
일단 업데이트 하는것 같으니 이 정도로 수정하면 좋을 것 같습니다.
```suggestion
const updateSellbiddingInfo = async (
```